stock_view_adj/views/stock_inventory_views.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="stock_inventory_form" model="ir.ui.view">
+        <field name="name">stock.inventory.form</field>
+        <field name="model">stock.inventory</field>
+        <field name="inherit_id" ref="stock.view_inventory_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//form/header/button[@name='action_done']" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </xpath>
+            <xpath expr="//form/header/button[@name='prepare_inventory']" position="after">
+                <button name="action_done" states="confirm" string="Validate Inventory" type="object" class="oe_highlight"/>
+            </xpath>
```

Comment:
@nakometal Can you try and see if this works, instead?
```
            <xpath expr="//form/header/button[@name='action_done']" position="attributes">
                <!-- remove stock.group_stock_manager assignment-->
                <attribute name="groups_id"></attribute>
            </xpath>
```

Reply:
@yostashiro It worked If it changes the attribute name `groups_id` to  `groups`.

Reply:
<xpath expr="//form/header/button[@name='action_done']" position="attributes">
                <!-- remove stock.group_stock_manager assignment-->
                <attribute name="groups"></attribute>
            </xpath>